.github/workflows/build-with-bal-test-native.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   call_stdlib_workflow:
     name: Run StdLib Workflow
     if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
-    uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-native-template.yml@main
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-graalvm-template.yml@main
```

Comment:
Need to rename the workflow file to - `build-with-bal-test-graalvm.yml`

Reply:
Renamed in [b121485](https://github.com/ballerina-platform/module-ballerina-xmldata/pull/502/commits/b12148549868f3439f598c87d503af5b3a2886fb)